change/@fluentui-react-89173b86-4f03-440a-93e8-23c4c3569f3b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26566&commitId=658a23ef9b28d7f02ce3ac6a5153dd2ef341e4d3&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.Multiselect - RTL.click.chromium.png|27| Changed|
|Dropdown.Multiselect - RTL.hover item.chromium.png|27| Changed|
|Dropdown.Multiselect.click.chromium.png|14| Changed|
|Dropdown.Multiselect.hover item.chromium.png|14| Changed|

</details>


<div id="vrtCommentfluentuiv8"/>

Reply:
These changes seem unrelated. I wonder if it's a problem with the baselines? Maybe pull master and try again?

Reply:
These exact screenshot diffs showed up in another PR where they were unrelated. I'm wondering if there's something going on with Pivot

Reply:
Both of those screenshots are supposed to be there, and appear to be correct. I think it may be an issue that existed earlier with Pivot screenshots.